frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/donut-chart.tsx

```diff
@@ -65,17 +74,34 @@ export const DiscoveryDonutChart: React.FC<DiscoveryDonutChartProps> = ({ state,
     <div className="ceph-ocs-install__chart-wrapper">
       <div className="ceph-ocs-install_capacity-header">Selected Capacity</div>
       <div className="ceph-ocs-install__stats">
-        <div>{pluralize(nodes.length, 'Node')}</div>
+        <div>
+          {state.filteredNodes.length ? (
+            <Button
+              component="a"
+              variant="link"
+              onClick={() => dispatch({ type: 'setShowNodeList', value: true })}
+              className="ceph-ocs-install__node-list-btn"
```

Comment:
```suggestion
              variant="link"
              onClick={() => dispatch({ type: 'setShowNodeList', value: true })}
              className="ceph-ocs-install__node-list-btn"
```
Any specific reason to make it an "anchor" tag ?
If you are doing that you also need to have `href` .